models/user.go

```diff
@@ -12,6 +13,22 @@ type User struct {
 	HashedPassword string
 }
 
+func (u *User) Keywords() orm.QuerySeter {
+	orm := orm.NewOrm()
+	return orm.QueryTable(Keyword{}).Filter("user_id", u.Id)
+}
+
+func (u *User) PaginatedKeywords(keywords orm.QuerySeter, offset int, sizePerPage int) ([]*Keyword, error) {
```

Comment:
While the DSL is nice `c.CurrentUser.Keywords()`, including all methods to retrieve keywords in the user model is not great in terms of separation of concerns and would not scale well when the application has lots of models. Instead, prefer moving the logic to get keyword filtered for a user in the keyword model itself. Or have an intermediary object such as query objects or service objects.

Reply:
I moved these functions to the `Keyword` model cc @gutakk